librz/bin/blang.c

```diff
@@ -1,66 +1,50 @@
-// SPDX-FileCopyrightText: 2018-2019 pancake <pancake@nopcode.org>
```

Comment:
@wargio i think we should keep this as well, WDYT?

Reply:
Yes

Reply:
not sure. i completely rewrote it. but yes we can